.github/workflows/chromatic.yml

```diff
@@ -39,7 +39,8 @@ jobs:
           node-version: "16.14.0"
       - name: Install Dependencies
         run: yarn install --frozen-lockfile
-        
+        uses: actions/checkout@v1
```

Comment:
checkout is there already in line 26

Reply:
But still this is not how you use checkout... How are you trying the workflow? by comment?